### ISSUE TITLE
Support serving csp_report from other domains than 0.0.0.0 and localhost

### ISSUE THREAD

**Issue by szimek** [rwjblue/ember-cli-content-security-policy#47]
I'm writing an app that uses WebRTC, which I need to serve it via valid HTTPS and I'm using [ngrok](https://ngrok.com/) for this. Currently when loading my app through ngrok domain, I'm getting:

```
Mixed Content: The page at 'https://xxxxxx.ngrok.io/' was loaded over HTTPS, but requested an insecure hyperlink auditing endpoint 'http://0.0.0.0:4200/csp-report'. This request has been blocked; the content must be served over HTTPS.
```

It looks like it can be easily fixed by adding `req.hostname` to a list of hosts [here](https://github.com/rwjblue/ember-cli-content-security-policy/blob/master/index.js#L38).

**Comment by zalun** [comment]
Have you provided a PR?

**Comment by jelhan** [comment]
Should be fixed by #109
